incubator-tools/cer_wer/cer_wer.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #82.    if __name__ == "__main__":
</pre><p>This isn't needed in a Notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Marking as done. Replaced the condition and called the main function without the condition.</p>